README.md

```diff
@@ -118,6 +133,9 @@ type `i` followed by the value name:
 
 ![elixir-i](https://cloud.githubusercontent.com/assets/14013616/20860322/3c01d984-b96d-11e6-8cc4-a46c8657f5b4.png)
 
+***
```

Comment:
I think these tend to look a bit weird with the double lines caused by the titles, but not a blocker at all. We don't have these anywhere else so I might suggest we remove them for consistency.

<img width="711" alt="screen shot 2017-02-16 at 18 51 46" src="https://cloud.githubusercontent.com/assets/4185328/23036001/20de1eb4-f479-11e6-8430-9198413ec490.png">